contrib/init/systemd/README.md

```diff
@@ -3,20 +3,37 @@
 > NOTE: These instructions are valid for Ubuntu versions using `systemd` init
 > system.
 
-Run the following commands:
+## TL;DR
+1. Get the latest [release](https://github.com/Azure/azurefile-dockervolumedriver/releases)
+2. Put the binary into `/usr/bin/azurefile-dockervolumedriver`
+3. Get the .default and .service files from the .tar.gz file and deploy them
+4. Reload systemd
+
+## In-depth walkthrough
+To ssh onto machines created with the `docker-machine` commands, you'll probably need to ssh via `docker-machine ssh [machinename]` to perform the following.
```

Comment:
I'm sorry but `docker-machine` has no place in this project. There are far too many ways to create a Docker engine on Azure and adding this will create more confusion.